src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +78,51 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            // Copy the trailing comments after the return statement
+            // Add them to our trailingCommentsHolder to use later
+            copyTrailingComments(returnStatement, trailingCommentsHolder, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+
+
+            // If there are trailing comments
+            if (trailingCommentsHolder && trailingCommentsHolder.emitNode && trailingCommentsHolder.emitNode.trailingComments) {

```

Comment:
It seems like you should be able to stop after copying the comments above. What case(s) is this trying to address?

Reply:
We probably also want to have something more general to do this like [copyComment](https://github.com/microsoft/TypeScript/blob/afc41f095d9337d57bee71f287ebf7ef39c8bdfb/src/services/utilities.ts#L2276) since we will probably have to do this in a bunch of places.

Reply:
Looks like this was to address the following cases. When I remove that functiona

#### `refactorAddOrRemoveBracesToArrowFunction25.ts`

```
Expected:
const a = (a: number) => a; /* trailing */

Actual:
const a = (a: number) => a;
```

#### `refactorAddOrRemoveBracesToArrowFunction26.ts`
```
Expected:
const a = (a: number) => /* leading */ a; /* trailing */

Actual:
const a = (a: number) => /* leading */ a;
```

#### `refactorAddOrRemoveBracesToArrowFunction27.ts`

```
Expected:
const b = (a: number) => /* leading */ a /* trailing */

Actual:
const b = (a: number) => /* leading */ a
```

#### `refactorAddOrRemoveBracesToArrowFunction28.ts`

```
Expected:
const a = (a: number) => a; /* trailing */ /* trailing */

Actual:
const a = (a: number) => a;
```

Reply:
> We probably also want to have something more general to do this like copyComments since we will probably have to do this in a bunch of places.

Agreed! Open to suggestions. How should we modify this?

Reply:
https://github.com/mjd0/material-remixer-remote-web

Reply:
So if you just make the changes you have on 81-84 but copy the comments to `returnStatement`, so

```ts
copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
copyTrailingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
```

we would take this:

```ts
const f = (a: number) /* a */ => /* b */ { /* c */
    /* d */
    return a /* e */; /* f */
};
```

and make this:

```ts
const f = (a: number) /* a */ => /* b */ /* c */ /* d */ a /* f */;
```

The `/* e */` comment gets eaten but probably because it belongs to the expression `a` rather than the `ReturnStatement`, but you can probably still copy with it one of the above methods. I wouldn't be heartbroken if that comment was just left out though because

1. We don't have to be perfect and the above is already much better than what we have, and
2. I think the placement of `/* e */` is ugly in the first place.

A change to refactoring generally involves changes to the AST manipulation, or changes to `emitter.ts`. Doing the above keeps us in the first which is probably the way you want to go in this PR. 

I think if you still need to make some changes to how the comment formatting is handled, I think you would want to that in `emitter.ts` (rather than copying its code here), and in a separate PR.